vynessa/wanderlust
# #155308324 Design trips page

#### What does this PR do?
- This PR contains the implementation for the design of the trips and trip details page

#### Description of Task to be completed?
- Design the trips page.
- Design a trip details page.

#### How should this be manually tested?
- Visit the trips page to see the new styling
- Select a trip card to see the details for the selected trip

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?
- #155308324

#### Screenshots (if appropriate)
- N/A

#### Questions
- N/A
